### PR TITLE
correcting jenkins compatibility error

### PR DESCRIPTION
**Pull request by joseims**
The jenkins is presenting some errors due to a compatibility problem, this just resolves it

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,5 @@ pytz
 pytest
 tzlocal
 mock
+urllib3==1.22
+
```
Comment (javanlacerda) [inline requirements.txt]: Empty line.

**Comment by joseims** [comment]
To whoever merge it, please squash and merge

**Comment by liliarsampaio** [comment]
Checks have failed and the branch has conflicts. Let me know when I can merge it.

**Comment by joseims** [comment]
This was corrected in another PR.

**Pull request closed without merging**